<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>929</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Video Processor</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: italic 15px;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>244</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>File Path / RTSP Url / Camera Index :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="videoEdit">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>460</y>
      <width>142</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startBtn">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>460</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>591</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Video">
     <attribute name="title">
      <string>Video</string>
     </attribute>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>40</y>
        <width>271</width>
        <height>201</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="upButton">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>260</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image:url(:/new/image/up.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="leftButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image:url(:/new/image/left.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="rightButton">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>300</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image:url(:/new/image/right.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="downButton">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>340</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image:url(:/new/image/down.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="cclockWiseButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>91</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image:url(:/new/image/cclock.png);
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="clockWiseButton">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>80</y>
        <width>91</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image:url(:/new/image/clock.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="extendButton">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>290</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image:url(:/new/image/extend.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="reduceButton">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>340</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image:url(:/new/image/reduce.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="recBtn">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>280</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px;
padding: 2px;
background-color: red;
font: bold italic 20px;
color:white;
</string>
      </property>
      <property name="text">
       <string>REC</string>
      </property>
     </widget>
     <widget class="QPushButton" name="LineBtn">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>330</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">padding: 2px;
background-color: gray;
font: bold italic 15px;
color:white;
</string>
      </property>
      <property name="text">
       <string>Linesearch</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="RGB">
     <attribute name="title">
      <string>RGB</string>
     </attribute>
     <widget class="QGraphicsView" name="graphicsView_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>541</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>340</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="RGBCheckLayout">
       <item>
        <widget class="QCheckBox" name="RedCheck">
         <property name="text">
          <string>Red</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="GreenCheck">
         <property name="text">
          <string>Green</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="BlueCheck">
         <property name="text">
          <string>Blue</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="chabtn">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>340</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>1channel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Detecting">
     <attribute name="title">
      <string>Detecting</string>
     </attribute>
     <widget class="QGraphicsView" name="graphicsView_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>541</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="detectBtn">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>340</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image:url(:/new/image/car.png);
color:white;
font: bold;</string>
      </property>
      <property name="text">
       <string>Detect</string>
      </property>
     </widget>
     <widget class="QPushButton" name="detectBtn2">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>340</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image:url(:/new/image/people.png);
color:red;
font: bold;</string>
      </property>
      <property name="text">
       <string>Detect</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>301</width>
      <height>431</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="HistogramLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Histogram</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGraphicsView" name="redHisto"/>
     </item>
     <item>
      <widget class="QGraphicsView" name="greenHisto"/>
     </item>
     <item>
      <widget class="QGraphicsView" name="blueHisto"/>
     </item>
    </layout>
   </widget>
   <zorder>label</zorder>
   <zorder>videoEdit</zorder>
   <zorder>startBtn</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>tabWidget</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
